templates/_components/index.html

```diff
@@ -107,7 +133,7 @@ <h4>Properties</h4>
 {% /breadcrumbs %}{% endverbatim breadcrumbs_container %}</code></pre>
       </div>
 
-      <div class="not-prose relative w-screen max-w-5xl left-1/2 -translate-x-1/2 p-8 bg-slate-200 border boder-slate-500 rounded">{% #breadcrumbs %}
+      <div class="not-prose relative w-screen max-w-5xl left-1/2 -translate-x-1/2 p-8 bg-slate-200 border border-slate-500 rounded">{% #breadcrumbs %}
```

Comment:
In other examples that you've introduced, like the lists and tables, you've put the main example first and then the details. Whereas for the breadcrumbs, you've put the main example last. I'm not sure which I prefer, but I think it should be consistent.

Reply:
Yup agreed, I've moved it first in 475d3be